### ISSUE TITLE
Bug: Wallet load wrong address history from cache

### ISSUE THREAD

**Issue by Joohansson** [linuxserver/nano-wallet#32]
Found a strange glitch. Are you able to replicate in your wallet?
1. Go to the login page
2. Type in /address/nano_xxx for a real address with some history
3. Press back button to come back to login page
4. Paste a seed (with empty history) and login
5. Now it will load the previous watched account instead of the account belonging to the seed as seen by the account history that is filled when it should be empty? But it seems the actual account used in the wallet is correct as seen in the receive window, so maybe just the history that is not cleared correctly.

**Comment by thelamer** [comment]
This can happen in a multitude of ways as your mounted stuff runs once on page load to determine what is being displayed, part of the reason we open the RO views in new windows . 

I am not sure there is an all encompassing solution to this, but also this does kind of go out of normal day to day use especially Mobile users we are targeting. Most normal users do not land on pages an manually modify URL entries.

**Comment by Joohansson** [comment]
Yeah ok. I agree it's kind of unique. It's strange though that it does not correct the history when you use the refresh button. Maybe that can be solved?
